contracts/StrategyCurve3crv.sol

```diff
@@ -68,20 +85,82 @@ abstract contract StrategyCurveBase is BaseStrategy {
         return stratName;
     }
 
-    function stakedBalance() public view returns (uint256) {
-        return gauge.balanceOf(address(this));
-    }
-
     function balanceOfWant() public view returns (uint256) {
         return want.balanceOf(address(this));
     }
 
+    function protectedTokens()
+        internal
+        view
+        override
+        returns (address[] memory)
+    {}
+
+    /* ========== SETTERS ========== */
+
+    // These functions are useful for setting parameters of the strategy that may need to be adjusted.
+
+    // This allows us to manually harvest with our keeper as needed
+    function setForceHarvestTriggerOnce(bool _forceHarvestTriggerOnce)
+        external
+        onlyAuthorized
+    {
+        forceHarvestTriggerOnce = _forceHarvestTriggerOnce;
+    }
+}
+
+contract StrategyCurve3crv is StrategyCurveBase {
+    /* ========== STATE VARIABLES ========== */
+    // these will likely change across different wants.
+
+    // Curve stuff
+    IGauge public constant gauge =
+        IGauge(0xB721Cc32160Ab0da2614CC6aB16eD822Aeebc101); // Curve gauge contract, most are tokenized, held by strategy
+
+    IGaugeFactory public constant gaugeFactory =
+        IGaugeFactory(0xabC000d88f23Bb45525E447528DBF656A9D55bf5);
+
+    // TODO: no longer required in the code. Confirm
+    // ICurveFi public constant curve =
```

Comment:
add_liquidity(...) to be performed by ySwaps so we no longer need the curve pool instance. Please confirm